docgen/parser/parser.go

```diff
@@ -54,6 +58,60 @@ func ParseFile(file *google_protobuf.FileDescriptorProto) (*File, error) {
 	return f, nil
 }
 
+func nestedDescriptorMessages(file google_protobuf.FileDescriptorProto, genFiles map[string]*google_protobuf.FileDescriptorProto) (map[string]*Message, error) {
```

Comment:
document, please

Reply:
Added a short description, please let me know if it needs more context